utils/env_posix.go

```diff
@@ -0,0 +1,13 @@
+//go:build !windows
```

Comment:
Optional: I don't think `posix` is a go keyword, but `unix` is. I think this could (should?) be named `env_unix.go` and then it doesn't need this tag here. If that was already tried and didn't work, or there's some other nuance I'm missing, then just ignore me.

Reply:
fairly sure the `_unix` extension doesn't work (per [this](https://github.com/golang/go/issues/51572) + iirc have been burned before)

'it's not an OS' is their rationale

Reply:
Ah, it's a "tag" but not a filename-recognized tag. That explains my confusion as well then, as I know it's a usable tag per https://pkg.go.dev/cmd/go#hdr-Build_constraints